src/Epicea/EpBehaviorCommentChange.class.st

```diff
@@ -54,6 +54,17 @@ EpBehaviorCommentChange >> behaviorAffectedName [
 	^ behavior name
 ]
 
+{ #category : 'accessing' }
+EpBehaviorCommentChange >> fileOutEntryOn: aWriteStream [ 
+
+	aWriteStream
+		nextPutAll: '(self class environment at: #';
```

Comment:
Aren's you missing the closing parenthesis here? And same for the other file out